src/nationalarchives/components/multi-select-search/_index.scss

```diff
@@ -0,0 +1,16 @@
+.tna-multi-select-search__list-container {
```

Comment:
Should the classes have `govuk` prefixed, as per the classes in the other component?

Reply:
Other departments seem to create their own namespace for components. I think `govuk` is reserved for the core GDS design system. 

https://design.homeoffice.gov.uk/components?name=Pagination#Pagination-example-html
https://service-manual.nhs.uk/design-system/components/text-input

Reply:
Would it be better to change the prefix to something else though? e.g. `tdr`?